packages/appstore/src/constants/wallets-intro-content-config.tsx

```diff
@@ -0,0 +1,53 @@
+import React from 'react';
+import WalletsImage from 'Assets/svgs/wallets';
+import { localize, Localize } from '@deriv/translations';
+
+type TWalletsIntroContent = {
+    image: JSX.Element;
+    title: string;
+    description: string;
+    bullets: string[];
+};
+
+const getWalletIsntroContent = (is_eu: boolean): TWalletsIntroContent[] => [
```

Comment:
```suggestion
const getWalletsIntroContent = (is_eu: boolean): TWalletsIntroContent[] => [
```
https://marketplace.visualstudio.com/items?itemName=streetsidesoftware.code-spell-checker

Reply:
Done 😃 👍🏼